BlueprintUI/Sources/Element/UIViewElement.swift

```diff
@@ -123,7 +123,7 @@ public extension UIViewElement {
             }
```

Comment:
Now that we can easily flow stuff through to `measurement`, it would be good to remove the static view cache and put it on each blueprint view instead.